ctapipe/core/telescope_component.py

```diff
@@ -110,7 +110,7 @@ def single_to_pattern(value):
         # make sure we only change things that are not already a
         # pattern tuple
         if (
-            not isinstance(value, tuple)
+            not isinstance(value, (list, tuple))
```

Comment:
This change makes me a bit nervous... The mechanics of deciding of something was already a default value or not where already pretty brittle before. What is the reason to  also check for a list here?

Reply:
Is not needed.  Left-over from the typing problem